pydis_site/apps/api/views.py

```diff
@@ -37,12 +37,15 @@ class RulesView(APIView):
 
     ## Routes
     ### GET /rules
-    Returns a JSON array containing the server's rules:
+    Returns a JSON array containing the server's rules
+    coupled with a list of keywords that will be used
+    when searching for particular rules upon using the
+    bot's `!rule` command
```

Comment:
Best just to describe what it does. How it is used is out of scope of the actual API.
```suggestion
    Returns a JSON array containing the server's rules
    and keywords relating to each rule. Example response:
```